src/app/components/common/courthouse/courthouse.component.html

```diff
@@ -1,16 +1,21 @@
 <label class="govuk-label" for="courthouse">Courthouse</label>
 <!-- autocomplete -->
-<div #courthouseAutocomplete id="courthouse-autocomplete-container" class="govuk-input--width-20">
-  <div *ngIf="!loaded && !loadError">Loading...</div>
-</div>
+<div
+  *ngIf="courthouses.length"
+  #courthouseAutocomplete
+  id="courthouse-autocomplete-container"
+  class="govuk-input--width-20"
+  [class.govuk-input--error]="isInvalid"
+></div>
 
-<!-- fall back to standard input if there is a load error-->
+<!-- fall back to standard input if there is no courthouses provided-->
 <input
-  *ngIf="loadError"
-  formControlName="courthouse"
+  *ngIf="!courthouses.length"
```

Comment:
What happens now if the courthouses error when loading?

Reply:
It falls back to a normal input